test/access/AccessControl.test.js

```diff
@@ -49,7 +49,7 @@ describe('AccessControl', function () {
     it('accounts can be granted a role multiple times', async function () {
       await this.accessControl.grantRole(ROLE, authorized, { from: admin });
       const receipt = await this.accessControl.grantRole(ROLE, authorized, { from: admin });
-      expectEvent(receipt, 'RoleGranted', { account: authorized, role: ROLE, operator: admin });
+      //await expectEvent.not.inTransaction(receipt.tx, AccessControlMock, 'RoleGranted');
```

Comment:
Is this commented out on purpose?

Reply:
Yes, the comment above explains why. Until the next version of test helpers is released we won't be able to use this helper.